scripts/west_commands/packages.py

```diff
@@ -95,6 +95,14 @@ def do_add_parser(self, parser_adder):
             "'--dry-run' to pip not to actually install anything, but print what would be.",
         )
 
+        pip_parser.add_argument(
+            "--ignore-venv",
```

Comment:
well it'd be more like `--ignore-venv-check`

Reply:
Sure, why not